package.json

```diff
@@ -17,6 +17,7 @@
     "build-main": "cross-env NODE_ENV=production node --trace-warnings -r babel-register ./node_modules/webpack/bin/webpack --config webpack.config.main.prod.js --progress --profile --colors",
     "build-renderer": "cross-env NODE_ENV=production node --trace-warnings -r babel-register ./node_modules/webpack/bin/webpack --config webpack.config.renderer.prod.js --progress --profile --colors",
     "start": "cross-env NODE_ENV=production electron ./app/",
+    "prestart": "npm run build",
```

Comment:
I can remove this, as it is documented in the readme. On the other hand I would expect `npm install` and `npm start` to "just work". But the readme clearly says you have to do `npm run build` before. Should I undo the change here?